packages/plugin/src/configs/schema-all.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * 🚨 IMPORTANT! Do not manually modify this file. Run: `yarn generate-configs`
+ */
+
+export default {
+  extends: ['plugin:@graphql-eslint/base', 'plugin:@graphql-eslint/schema-recommended'],
+  rules: {
+    '@graphql-eslint/alphabetize': [
+      'error',
+      {
+        fields: ['ObjectTypeDefinition', 'InterfaceTypeDefinition', 'InputObjectTypeDefinition'],
+        values: ['EnumTypeDefinition'],
+        arguments: ['FieldDefinition', 'Field', 'DirectiveDefinition', 'Directive'],
+      },
+    ],
+    '@graphql-eslint/input-name': 'error',
+    '@graphql-eslint/no-root-type': 'off',
```

Comment:
Why setting these explicitly to 'off'? I'm having trouble seeing a scenario where folks would want extending their config with this ruleset to turn off these rules for them.

Reply:
This rule requires a provided option, I have set to `off` to explicitly show which rules are disabled in the `all` config, if the user will want to enable this rule it's totally fine

Reply:
Got it. Sounds like it's a part of the way things are generated.

Low impact, but setting explicitly to 'off' may inadvertently place requirements on folks' extends chains. E.g.:

```
    ...
    extends: [
        'prettier/@typescript-eslint',
        'plugin:@justin-eslint',
        'plugin:@graphql-eslint/schema-all'
  ],
```

If `@justin-eslint` sets `no-root-type`, extending it before `@graphql-eslint/schema-all` will result in the rule being unset, which I think might be a surprise for folks.